src/domain/LottoGame.js

```diff
@@ -0,0 +1,162 @@
+import Lotto from "./Lotto.js";
+import LottoPrize from "./LottoPrize.js";
+
+class LottoPurchaseError extends Error {
+  constructor(purchaseUnit) {
+    super(`구입 금액은 ${purchaseUnit}원 단위로 입력해야 합니다.`);
+  }
+}
+
+class InvalidWinningNumberLength extends Error {
+  constructor(selectionCount) {
+    super(
+      `당첨 번호는 ${selectionCount}개, 보너스 번호는 1개를 입력해야 합니다.`
+    );
+  }
+}
+
+class InvalidLottoDrawNumber extends Error {
+  constructor(min, max) {
+    super(`당첨 번호는 ${min} 이상 ${max} 이하 숫자여야 합니다.`);
+  }
+}
+
+export default class LottoGame {
+  static PRICE_PER_LOTTO = 1000;
+  static LOTTO_SELECTION_COUNT = 6;
+  static NUMBER_MIN_RANGE = 1;
+  static NUMBER_MAX_RANGE = 45;
+  static PRIZE_CONDITIONS = [
+    {
+      requiredMatchCount: 3,
+      bonusMatched: false,
+      prizeMoney: 5000,
+    },
+    {
+      requiredMatchCount: 4,
+      bonusMatched: false,
+      prizeMoney: 50000,
+    },
+    {
+      requiredMatchCount: 5,
+      bonusMatched: false,
+      prizeMoney: 1500000,
+    },
+    {
+      requiredMatchCount: 5,
+      bonusMatched: true,
+      prizeMoney: 30000000,
+    },
+    {
+      requiredMatchCount: 6,
+      bonusMatched: false,
+      prizeMoney: 2000000000,
```

Comment:
위에서 언급했던 numeric sperator는 숫자가 클수록 더더욱 유용하니 꼭 적용해보시면 좋겠네요 ㅎㅎ
```suggestion
      prizeMoney: 2_000_000_000,
```